languages/csharp/exercises/concept/lists/ListsTests.cs

```diff
@@ -0,0 +1,108 @@
+using Xunit;
+
+public class ListsTests
+{
+    [Fact]
+    public void NewList()
+    {
+        Assert.Empty(Languages.NewList());
+    }
+
+    [Fact(Skip = "Remove this Skip property to run this test")]
+    public void ExistingList()
+    {
+        Assert.Equal(new string[] {"C#", "Clojure", "Elm"}, Languages.GetExistingLanguages());
```

Comment:
Should we perhaps use a `List<string>` instance to compare against? For a CLI-based student that will see the test suite, we are now comparing a `List<string>` to a `string[]`, which may not be obvious why that works.

The main downside is that one would have to create a list, and we can't use the collection initializer at this point, right? So that would mean doing something like this:

```suggestion
        var expected = new List<string>();
        expected.Add("C#");
        expected.Add("Clojure");
        expected.Add("Elm");
        Assert.Equal(expected, Languages.GetExistingLanguages());
```

More tedious but maybe more intuitive for CLI-based students.